From 957e4949716644fe7145bb7cdad6a77907502e2a Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 25 Oct 2023 14:45:32 +0200
Subject: [PATCH] ARM: dts: stm32: add spi6 SCMI clock / reset for stm32mp15
 boards

Update the clock / reset properties for spi6 node in case of
scmi based boards.

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: I72956c7f681567ef6ec48db77dff951838521d53
---
 arch/arm/boot/dts/st/stm32mp157a-ed1-scmi.dtsi | 5 +++++
 arch/arm/boot/dts/st/stm32mp157c-ed1-scmi.dtsi | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/st/stm32mp157a-ed1-scmi.dtsi b/arch/arm/boot/dts/st/stm32mp157a-ed1-scmi.dtsi
index 62a0117f050b..283f63f85d15 100644
--- a/arch/arm/boot/dts/st/stm32mp157a-ed1-scmi.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp157a-ed1-scmi.dtsi
@@ -84,6 +84,11 @@ &rtc {
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
 
+&spi6 {
+	clocks = <&scmi_clk CK_SCMI_SPI6>;
+	resets = <&scmi_reset RST_SCMI_SPI6>;
+};
+
 &usart1 {
 	clocks = <&scmi_clk CK_SCMI_USART1>;
 };
diff --git a/arch/arm/boot/dts/st/stm32mp157c-ed1-scmi.dtsi b/arch/arm/boot/dts/st/stm32mp157c-ed1-scmi.dtsi
index 0ae22edb2537..a26929e6966b 100644
--- a/arch/arm/boot/dts/st/stm32mp157c-ed1-scmi.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp157c-ed1-scmi.dtsi
@@ -89,6 +89,11 @@ &rtc {
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
 
+&spi6 {
+	clocks = <&scmi_clk CK_SCMI_SPI6>;
+	resets = <&scmi_reset RST_SCMI_SPI6>;
+};
+
 &usart1 {
 	clocks = <&scmi_clk CK_SCMI_USART1>;
 };
-- 
2.39.5

